Makefile

```diff
@@ -84,3 +84,6 @@ docker-internal-linkcheck:
 container-internal-linkcheck: link-checker-image-pull
 	$(CONTAINER_RUN) $(CONTAINER_IMAGE) hugo --config config.toml,linkcheck-config.toml --buildFuture
 	$(CONTAINER_ENGINE) run --mount type=bind,source=$(CURDIR),target=/test --rm wjdp/htmltest htmltest
+
+partial-serve:
+	 hugo server --buildFuture --contentDir content/$(LANG)
```

Comment:
@irvifa, could we add `container-partial-serve` as well?

Reply:
Yep we can do that. Addressed in current commit.